security/form_login_setup.rst

```diff
@@ -279,6 +302,8 @@ a traditional HTML form that submits to ``/login``:
                     guard:
                         authenticators:
                             - App\Security\LoginFormAuthenticator
+                    logout:
+                        path: app_logout
 
     .. code-block:: xml
 
```

Comment:
XML and PHP config blocks should be updated as well, but I'm not sure about the correct syntax as I'm not using them